crates/telio-nurse/src/qos.rs

```diff
@@ -406,13 +407,27 @@ impl Analytics {
                     continue;
                 }
 
-                let (pk, endpoint) = (node.public_key, node.endpoint);
+                let (pk, mut ip_address) = (node.public_key, node.ip_address);
                 let pinger = Arc::clone(&self.ping_backend);
                 let ping_channel_tx = ping_channel_tx.clone();
 
                 tokio::spawn(async move {
                     if let Some(pinger) = &*pinger {
-                        Box::pin(pinger.perform((pk, endpoint), ping_channel_tx)).await;
+                        let mut dpr = Box::pin(pinger.perform((pk, ip_address))).await;
+
+                        // If we're pinging VPN address 100.64.0.1 and it's not
+                        // reachable, ping 10.5.0.1 instead
+                        if ip_address.target.0 == Some(Ipv4Addr::new(100, 64, 0, 1)) {
+                            if let Some(results_v4) = &dpr.v4 {
+                                if results_v4.successful_pings == 0 {
+                                    telio_log_warn!("VPN was not reachable through 100.64.0.1, trying 10.5.0.1 instead.");
```

Comment:
We will have a lot of these now maybe let's have it as debug.

Reply:
changed to debug log level